hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -287,6 +289,11 @@ public final class Hedera implements SwirldMain, PlatformStatusChangeListener {
      */
     @Nullable
     private CompletableFuture<Bytes> initialStateHashFuture;
+    /**
+     * The roster used on genesis. This is set in {@link ServicesMain} on platform initialization.
+     * This is used to construct the {@link GenesisNetworkInfo} object.
+     */
+    private Roster genesisRoster;
```

Comment:
Only for genesis, or also for reconnect etc.?

Reply:
Removed this.